src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms ease;
+
+    &:hover {
+      color: #00acdc;
```

Comment:
Consider using a variable for the color `#00acdc` to maintain consistency and improve readability. Example: `$hover-color: #00acdc;` 😊